apps/gitness/src/pages/repo-sandbox/repo-sandbox-summary.tsx

```diff
@@ -176,93 +208,106 @@ export const RepoSandboxSummaryList: React.FC = () => {
+                        <Button variant="outline">
+                          Add file&nbsp;&nbsp;
+                          <Icon name="chevron-down" size={11} className="chevron-down" />
+                        </Button>
+                      </DropdownMenuTrigger>
+                      <DropdownMenuContent align="start">
+                        <DropdownMenuItem
+                          key={'create-file'}
+                          onClick={() => {
+                            navigate(`/spaces/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranch}/~/`)
+                          }}>
+                          + Create New File
+                        </DropdownMenuItem>
+                      </DropdownMenuContent>
+                    </DropdownMenu>
+                    <CloneRepoDialog
+                      sshUrl={repository?.git_ssh_url ?? 'could not fetch url'}
+                      httpsUrl={repository?.git_url ?? 'could not fetch url'}
+                      handleCreateToken={handleCreateToken}
+                    />
+                  </ButtonGroup.Root>
+                </ListActions.Right>
+              </ListActions.Root>
+              <Spacer size={5} />
+              {renderListContent()}
+              <Spacer size={12} />
+              <StackedList.Root>
+                <StackedList.Item isHeader disableHover>
+                  <StackedList.Field title={<Text color="tertiaryBackground">README.md</Text>} />
+                </StackedList.Item>
+                <StackedList.Item disableHover>
+                  <MarkdownViewer source={decodedReadmeContent || ''} />
+                </StackedList.Item>
+              </StackedList.Root>
+            </SandboxLayout.Content>
+          </SandboxLayout.Column>
+          <SandboxLayout.Column>
+            <SandboxLayout.Content className="pl-0">
+              <RepoSummaryPanel
+                title="Summary"
+                details={[
+                  {
+                    id: '0',
+                    name: 'Commits',
+                    count: default_branch_commit_count || 0,
+                    iconName: 'tube-sign'
+                  },
+                  {
+                    id: '1',
+                    name: 'Branches',
+                    count: branch_count || 0,
+                    iconName: 'branch'
+                  },
+                  {
+                    id: '2',
+                    name: 'Tags',
+                    count: tag_count || 0,
+                    iconName: 'tag'
+                  },
+                  {
+                    id: '3',
+                    name: 'Open pull requests',
+                    count: pull_req_summary?.open_count || 0,
+                    iconName: 'open-pr'
+                  }
+                ]}
+              />
+            </SandboxLayout.Content>
+          </SandboxLayout.Column>
+        </SandboxLayout.Columns>
+      </SandboxLayout.Main>
+      {createdTokenData && (
```

Comment:
why do we need this condition?

Reply:
in the case where `token-creation` failed, we do not open the dialog.

Reply:
In that case we should be able to do:
```
open={createdTokenData && successTokenDialog}
```